README.md

```diff
@@ -395,6 +393,8 @@ This project welcomes contributions and suggestions.
 **Here are some 
 [code standards](docs/developer/code_standard.rst) when you submit a pull request.**
 
+If you want to modify the qlib document, you can follow the following figure.
+https://github.com/demon143/qlib/blob/main/docs/_static/img/change%20doc.gif
```

Comment:
You can embed a URL from the official repo instead of your personal one.

Reply:
For example

This can display the image directly instead of forcing users to click the link
```
<p align="center">
  <img src="http://fintech.msra.cn/images_v060/logo/1.png" />
</p>
```